src/event/block/BlockSproutEvent.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace pocketmine\event\block;
+
+use pocketmine\block\Block;
+use pocketmine\event\Cancellable;
+use pocketmine\event\CancellableTrait;
+use pocketmine\world\BlockTransaction;
+
+/**
+ * Called when Saplings or Bamboo grow.
```

Comment:
```suggestion
 * Called when objects such as Saplings or Bamboo grow.
```
This might be applied to other things in the future (e.g. kelp, chorus plants, etc).